packages/ui/test/working-groups/components/CreateOpeningButton.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import { CreateOpeningButton } from '@/working-groups/components/CreateOpeningButton'
+
+import { getButton } from '../../_helpers/getButton'
+
+const group = 'storageWorkingGroup'
+const newOpeningButtonTitle = 'Add opening'
+
+describe('UI: CreateOpeningButton', () => {
+  it.skip('Lead sees CreateOpeningButton', async () => {
+    // use lead worker
+
+    renderComponent()
+
+    const button = await getButton(newOpeningButtonTitle)
+    expect(button).not.toBeDisabled()
+  })
+
+  it.skip('CreateOpening modal', async () => {
+    // use lead worker
+
+    renderComponent()
+
+    const button = await getButton(newOpeningButtonTitle)
+    expect(button).not.toBeDisabled()
+  })
+
+  const renderComponent = () => render(<CreateOpeningButton group={group} />)
+})
```

Comment:
@traumschule started adding tests which is indeed needed here. But it could be painful to add unit tests for this flow. Instead in another PR could you add an opening tab story under "Pages/Working Groups/Openings" please ? Once this other PR is merged you could add the tests for this modal in this new story.